server/src/main/java/org/elasticsearch/action/admin/indices/recovery/RecoveryRequest.java

```diff
@@ -91,14 +82,14 @@ public void activeOnly(boolean activeOnly) {
     @Override
     public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
-        out.writeBoolean(detailed);
+        out.writeBoolean(details);
```

Comment:
when you remove this, watch out for BWC aspects - see https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/action/admin/cluster/storedscripts/GetStoredScriptRequest.java#L59 as an example.

Reply:
Here I just changed the name of variable (to be the same as flag name), the value is always a boolean, so we don't need to do some bwc changes ?